pkg/minikube/assets/vm_assets.go

```
@@ -139,24 +139,34 @@ func (m *MemoryAsset) Read(p []byte) (int, error) {
 }
 
 func CopyFileLocal(f CopyableFile) error {
-	os.MkdirAll(f.GetTargetDir(), os.ModePerm)
+	err := os.MkdirAll(f.GetTargetDir(), os.ModePerm)
+	if err != nil {
+		return errors.Wrapf(err, "error making dirs for %s", f.GetTargetDir())
+	}
 	targetPath := filepath.Join(f.GetTargetDir(), f.GetTargetName())
-	os.Remove(targetPath)
-	target, err := os.Create(targetPath)
-	defer target.Close()
+	if _, err := os.Stat(targetPath); err == nil {
+		err = os.Remove(targetPath)
+		if err != nil {
+			return errors.Wrapf(err, "error removing file %s", targetPath)
+		}
 
+	}
+	target, err := os.Create(targetPath)
```

Looks like you're ignoring err here?